packages/react-native-web/src/exports/Image/__tests__/index-test.js

```diff
@@ -21,6 +21,9 @@ describe('components/Image', () => {
   beforeEach(() => {
     ImageUriCache._entries = {};
     window.Image = jest.fn(() => ({}));
+    ImageLoader.load = jest.fn().mockImplementation((src, onLoad, onError) => {
+      onLoad(src);
+    });
```

Comment:
Some tests were modifying the behavior of the `ImageLoader.load` method, since there's no cleanup 
the behavior would remain to the last `jest.fn().mockImplementation` assigned to the method

It's more consistent to have the behavior predefined uniformly before each test, while we still let some tests have a custom implementation